src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/SyncCloudContent.targets

```diff
@@ -0,0 +1,31 @@
+<Project ToolsVersion="12.0" DefaultTargets="DownloadBlobsFromAzureTargets" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <UsingTask TaskName="DownloadFromAzure" AssemblyFile="$(ToolsDir)net45/Microsoft.DotNet.Build.CloudTestTasks.dll"/>
```

Comment:
Instead of $(ToolsDir), use $(BuildToolsTaskDir) which will use the net45 version of the dll for Windows the build and Core compiled version on non-Windows platform builds.